### PR TITLE
Update typo, language, and links in Preservation Planning

### PR DESCRIPTION
**Pull request by ablwr**

### CODE REVIEW

File: user-manual/preservation/preservation-planning.rst [hunk 1]
```diff
@@ -398,8 +398,7 @@ Writing commands
 Identification commands
 """""""""""""""""""""""
 
-Identification commands are very simple to write, though they require some
-familiarity with Unix scripting.
+Identification commands require some familiarity with Unix scripting.
```
Comment (sallain) [inline user-manual/preservation/preservation-planning.rst:400]: I like this change a lot :sparkle:

File: user-manual/preservation/preservation-planning.rst [hunk 2]
```diff
@@ -70,8 +70,8 @@ When creating a new format version, the following fields are available:
   choose "14".
 
 * PRONOM ID - The specific format version's unique identifier in
-  `PRONOM <http://www.nationalarchives.gov.uk/PRONOM/Default.aspx>`_, the UK
-  National Archives's format registry. This is optional, but highly recommended.
+  `PRONOM <PRONOM_>`_, the UK National Archives's format registry. This is 
+  optional, but highly recommended.
   Many identification and characterization tools, use PRONOM ID's to recognize
```
Comment (sallain) [inline user-manual/preservation/preservation-planning.rst]: While you're in here can you fix this comma splice? :slightly_smiling_face:

**Review by sallain — APPROVED** [review]
LGTM!

**Comment by sevein** [comment]
👏 👏 👏

**Review by sevein — APPROVED** [review]

File: user-manual/preservation/preservation-planning.rst (hunk 2 above)
Comment (ablwr) [reply user-manual/preservation/preservation-planning.rst]: Done!

**Comment by ablwr** [comment]
I merged and see changes in the site, so I think I can now close this branch? Any other process steps I missed?

**Comment by ablwr** [comment]
This did not turn purple though. 🤔

**Comment by ablwr** [comment]
Good to go, discussed offline. I think I understand the issue.

**Pull request closed without merging**